harness/o.n.insane/src/org/netbeans/insane/impl/InsaneEngine.java

```diff
@@ -51,23 +52,23 @@ public InsaneEngine(Filter f, Visitor v, boolean analyzeStatic) {
 
     public InsaneEngine(ObjectMap backend, Filter f, Visitor v, boolean analyzeStatic) {
         objects = backend;
-        filter = f == null ? ScannerUtils.noFilter() : f;
+        filter = (f != null ? f : ScannerUtils.noFilter());
```

Comment:
I don't see the improvement, on the contrary, the new form introduces noise.

Reply:
reverted